### PR TITLE
Dev QOL for Pytest

### PR DESCRIPTION
**Pull request by GameDungeon**
## Motivation

A nicer coat of paint on pytest.

image

Also moved community colors into constants.

## Checklist
- [X] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [X] I have written a descriptive PR title (see top of PR template for examples)
- [X] I have written a changelog entry for the PR or deem it unnecessary

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Comment by GameDungeon** [comment]
Easy Review

File: manim/communitycolors.py [hunk 1]
```diff
@@ -1,7 +0,0 @@
-"""The colormap of manim community"""
```
Comment (naveen521kk) [inline manim/communitycolors.py:1]: it is here on purpose...
until we have something like themes, see the PR where it was created to get the info.

**Comment by naveen521kk** [comment]
> A nicer coat of paint on pytest.

Why should I even need it. It's extra dependency occupying precious disk space.

**Comment by kilacoda** [comment]
Yeah I don't see the need for it either, the tests are pretty readable as they are.

File: manim/communitycolors.py (hunk 1 above)
Comment (naveen521kk) [reply manim/communitycolors.py:1]: https://github.com/ManimCommunity/manim/pull/892

Comment (GameDungeon) [reply manim/communitycolors.py:1]: A poll on the discord server getting quite a few responses unanimously agreed with me.

**Pull request closed without merging**